go/pkg/db/schema.sql

```diff
@@ -385,45 +351,30 @@ CREATE TABLE `acme_users` (
 	CONSTRAINT `acme_users_id` PRIMARY KEY(`id`)
 );
 
-CREATE TABLE `hostname_routes` (
+CREATE TABLE `domains` (
 	`id` varchar(256) NOT NULL,
 	`workspace_id` varchar(256) NOT NULL,
-	`project_id` varchar(256) NOT NULL,
-	`hostname` varchar(256) NOT NULL,
-	`deployment_id` varchar(256) NOT NULL,
-	`is_enabled` boolean NOT NULL DEFAULT true,
+	`project_id` varchar(256),
+	`deployment_id` varchar(256),
+	`domain` varchar(256) NOT NULL,
+	`type` enum('custom','wildcard') NOT NULL,
 	`created_at` bigint NOT NULL,
 	`updated_at` bigint,
-	CONSTRAINT `hostname_routes_id` PRIMARY KEY(`id`),
-	CONSTRAINT `hostname_idx` UNIQUE(`hostname`)
+	CONSTRAINT `domains_id` PRIMARY KEY(`id`)
 );
 
-CREATE TABLE `domain_challenges` (
+CREATE TABLE `acme_challenges` (
 	`id` bigint unsigned AUTO_INCREMENT NOT NULL,
-	`workspace_id` varchar(255) NOT NULL,
-	`domain_id` varchar(255) NOT NULL,
-	`token` varchar(255),
-	`authorization` varchar(255),
-	`status` enum('waiting','pending','verified','failed','expired') NOT NULL DEFAULT 'pending',
-	`type` enum('http-01','dns-01','tls-alpn-01') NOT NULL DEFAULT 'http-01',
-	`created_at` bigint NOT NULL,
-	`updated_at` bigint,
-	`expires_at` bigint unsigned,
-	CONSTRAINT `domain_challenges_id` PRIMARY KEY(`id`),
-	CONSTRAINT `domainIdWorkspaceId_idx` UNIQUE(`domain_id`,`workspace_id`)
-);
-
-CREATE TABLE `domains` (
-	`id` varchar(255) NOT NULL,
-	`workspace_id` varchar(255) NOT NULL,
-	`project_id` varchar(255) NOT NULL,
-	`domain` varchar(255) NOT NULL,
-	`type` enum('custom','generated') NOT NULL DEFAULT 'generated',
-	`subdomain_config` json,
+	`workspace_id` varchar(256) NOT NULL,
+	`domain_id` varchar(256) NOT NULL,
+	`token` varchar(256) NOT NULL,
+	`challenge_type` enum('HTTP-01','DNS-01') NOT NULL,
```

Comment:
can we just name this type 😅

Reply:
Just ‘type’ that this is the challenge type is implied by the table name

Reply:
fair enough